bindings/swift/iota_wallet_ffi.h

```diff
@@ -13,10 +13,14 @@ typedef struct iota_wallet_handle iota_wallet_handle_t;
 
 typedef void (*Callback)(const char* response, const char* error, void* context);
 
+// Initialise the fern logger. Level filter defaults to Debug if omitted
+extern int8_t iota_init_logger(const char* file_name, const char* level_filter);
 extern iota_wallet_handle_t* iota_initialize(const char* manager_options, char* error_buffer, size_t error_buffer_size);
 extern void iota_destroy(iota_wallet_handle_t*);
 extern void iota_send_message(iota_wallet_handle_t* wallet_handle, const char* message, Callback callback, void* context);
 extern int8_t iota_listen(iota_wallet_handle_t* wallet_handle, const char* event_types, Callback callback, void* context, char* error_buffer, size_t error_buffer_size);
+extern int8_t iota_clear_listeners(iota_wallet_handle_t* wallet_handle, const char* event_types, Callback callback, void* context, char* error_buffer, size_t error_buffer_size);
```

Comment:
I'm not a very big fan of the prefixes. We don't have that in the other bindings do we ?

Reply:
No, but this is a C thing, as there are no namespaces, this is how most people make their methods